README.md

```diff
@@ -21,26 +21,52 @@ If you don't happen to know what a carbon credit is:
 
 * Set pg and sequel.pg to use libpq
 
-       bundle config --local build.pg --with-opt-dir="/opt/homebrew/opt/libpq"
-       bundle config --local build.sequel_pg --with-opt-dir="/opt/homebrew/opt/libpq"
+        bundle config --local build.pg --with-opt-dir="/opt/homebrew/opt/libpq"
+        bundle config --local build.sequel_pg --with-opt-dir="/opt/homebrew/opt/libpq"
```

Comment:
I'm curious about this recommendation. Can you explain the benefit of setting this up?